templates/install/bundles/amq-online/050-Deployment-enmasse-operator.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       serviceAccountName: enmasse-operator
       containers:
       - name: controller
-        image: amq7/amq-online-1-controller-manager:dev
+        image: registry.redhat.io/amq7/enmasse-controller-manager:dev
```

Comment:
Should this be amq-online-1-controller-manager instead of enmasse-controller-manager?

Reply:
Indeed, thanks!

Reply:
Actually this needs to be fixed in a different repo, so it will correct itself once the build has run.